recently commit is broken my pontoon instance

Running webpack...
Hash: 51288b18ff313d4c8812
Version: webpack 4.46.0
Time: 9178ms
Built at: 09/30/2021 4:52:48 PM
                       Asset      Size  Chunks                   Chunk Names
               tag_admin.css  9.34 KiB       0  [emitted]        tag_admin
           tag_admin.css.map  12.1 KiB       0  [emitted] [dev]  tag_admin
    tag_admin.entry.chunk.js   183 KiB       0  [emitted]        tag_admin
tag_admin.entry.chunk.js.map   726 KiB       0  [emitted] [dev]  tag_admin
Entrypoint tag_admin = tag_admin.css tag_admin.entry.chunk.js tag_admin.css.map tag_admin.entry.chunk.js.map
[16] ./pontoon/static/js/widgets/errors.css 39 bytes {0} [built]
[17] ./pontoon/static/js/tags/admin/tag-resources.css 39 bytes {0} [built]
[18] ./pontoon/static/js/tags/admin/index.js + 19 modules 330 KiB {0} [built]
     | ./pontoon/static/js/tags/admin/index.js 419 bytes [built]
     | ./pontoon/static/js/tags/admin/button.js 1.2 KiB [built]
     | ./pontoon/static/js/tags/admin/manager.js 1.55 KiB [built]
     | ./pontoon/static/js/tags/admin/search.js 1.22 KiB [built]
     | ./pontoon/static/js/tags/admin/table.js 1.01 KiB [built]
     | ./pontoon/static/js/widgets/errors.js 1.29 KiB [built]
     | ./pontoon/static/js/utils/data.js 2.02 KiB [built]
     | ./pontoon/static/js/widgets/columns.js 2.16 KiB [built]
     | ./pontoon/static/js/widgets/tables/checkbox.js 4.77 KiB [built]
     | ./pontoon/static/js/utils/ajax.js 4.7 KiB [built]
     | ./pontoon/static/js/widgets/checkbox.js 808 bytes [built]
     | ./pontoon/static/js/utils/strip.js 618 bytes [built]
     | ./pontoon/static/js/utils/csrf.js 112 bytes [built]
     |     + 7 hidden modules
    + 19 hidden modules

Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!node_modules/react-table/react-table.css:
    Entrypoint mini-css-extract-plugin = *
       2 modules
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!pontoon/static/js/tags/admin/tag-resources.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./pontoon/static/js/tags/admin/tag-resources.css 1.05 KiB {0} [built]
        + 1 hidden module
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!pontoon/static/js/widgets/errors.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./pontoon/static/js/widgets/errors.css 385 bytes {0} [built]
        + 1 hidden module

Building frontend resources...
/tmp/build_f6fc385f/frontend /tmp/build_f6fc385f
> frontend@0.1.0 build /tmp/build_f6fc385f/frontend
> react-scripts build
Creating an optimized production build...
Failed to compile.
/tmp/build_f6fc385f/frontend/src/index.tsx
TypeScript error in /tmp/build_f6fc385f/frontend/src/index.tsx(22,19):
Property 'locale' does not exist on type 'typeof TimeAgo'.  TS2339
    20 | // make TimeAgo internationalized and initialize all locales here.
    21 | // See: https://www.npmjs.com/package/react-time-ago
  > 22 | JavascriptTimeAgo.locale(en);
       |                   ^
    23 | 
    24 | ReactDOM.render(
    25 |     <Provider store={store}>
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! frontend@0.1.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the frontend@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /app/.npm/_logs/2021-09-30T16_54_43_664Z-debug.log
Collecting static files...
bin/post_compile: line 21: ./manage.py: No such file or directory
bin/post_compile: line 22: ./manage.py: No such file or directory
Current commit being deployed: 
3e5c97955902aa78576a458991bc36624a4cddf9
bin/post_compile: line 29: static/revision.txt: No such file or directory
-----> Discovering process types
       Procfile declares types -> web, worker
-----> Compressing...
       Done: 254.2M
-----> Launching...
       Released v76
       https://-redacted-.herokuapp.com/ deployed to Heroku

so maybe it's frontend things, so idk what happen. it's getting internal sever error in pontoon intance. my pontoon git is broken or commit is broken?

--- a/frontend/package.json
+++ b/frontend/package.json
@@ -14,6 +14,7 @@
     "highcharts": "9.0.0",
     "highcharts-react-official": "2.2.2",
     "html-react-parser": "0.13.0",
+    "javascript-time-ago": "2.3.8",
     "jest-each": "24.5.0",
     "linkify-it": "2.1.0",
     "lodash.clonedeep": "4.5.0",

--- a/frontend/src/index.tsx
+++ b/frontend/src/index.tsx
@@ -3,8 +3,6 @@ import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 
 import { ConnectedRouter } from 'connected-react-router';
-// javascript-time-ago is installed automatically with react-time-ago
-// See: https://www.npmjs.com/package/react-time-ago
 import JavascriptTimeAgo from 'javascript-time-ago';
 import en from 'javascript-time-ago/locale/en';
 

